### ISSUE TITLE
Memory consumption in open/close

### ISSUE THREAD

**Issue by Hugoxz** [rotorgames/Rg.Plugins.Popup#69]
Hello,

I have problems with memory consumption in open and close the popup many times and with my popup that I display a ListView, it throw exception of OutOfMemory. You guys have this problem too?
Here is my code for now how I using the popup to solve this problem:

```C#
public partial class MyPage : ContentPage
{
	private static WeakReference<MyPopup> refPopup;

	private void OnClick(object sender, EventArgs e)
	{
		MyPopup popup = null;

		if (refPopup == null || !refPopup.TryGetTarget(out popup))
		{
			popup = new MyPopup();
			refPopup = new WeakReference<MyPopup>(popup);
		}

		popup.SomeProp = "Value";

		await Navigation.PushPopupAsync(popup);
		// await Navigation.PushPopupAsync(popup, false); // will not work at second opening
	}
}
```

But how I commented in code, if you set animated to false, it will not work at second openning.
What do you think about?

**Comment by rotorgames** [comment]
@Hugoxz Hello. Thank you for question. Please wait, I will look this error.

**Comment by rotorgames** [comment]
This issue was fixed in v1.1.0-pre1. Please test it and tell me about your tests. Thank you.
